src/redux/walletconnect.js

```diff
@@ -149,13 +150,19 @@ export const walletConnectOnSessionRequest = (uri, callback) => async (
               dappName,
               dappUrl,
             });
-          } else {
+          } else if (!timedOut) {
             await dispatch(walletConnectRejectSession(peerId, walletConnector));
-            callback && callback('reject', dappScheme);
+            callback?.('reject', dappScheme);
             analytics.track('Rejected new WalletConnect session', {
               dappName,
               dappUrl,
             });
+          } else {
+            callback?.('timedOut', dappScheme);
+            analytics.track('New WalletConnect session time out', {
+              dappName,
+              dappUrl,
```

Comment:
Can we add the bridge url here? I think that would be super useful

Reply:
💯